rct229/rules/section22/section22rule27.py

```diff
@@ -0,0 +1,69 @@
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+
+APPLICABLE_SYS_TYPES = [
```

Comment:
The `get_baseline_system_types` is merged. Please update the `APPLICABLE_SYS_TYPES` list and the `is_applicable` function.

Reply:
Fixed.